Gopkg.lock

```diff
@@ -1140,12 +1139,13 @@
   version = "v4.3.0"
 
 [[projects]]
-  digest = "1:8a0efb153cc5b7e0e129d716834217be483e2b326e72f3dcca8b03cd3207e9e4"
+  digest = "1:bcee186946d33b2e3befed808ee6ffe73387345807187e51790e3e148b5344c2"
```

Comment:
Why update this dep as it seems to not be used in this PR ?

Reply:
Yeah, that was my mistake, I wanted to do that in other PR 😁

Reply:
fixed